src/components/PokemonPage.js

```diff
@@ -1,21 +1,48 @@
-import React from "react";
-import PokemonCollection from "./PokemonCollection";
-import PokemonForm from "./PokemonForm";
-import Search from "./Search";
-import { Container } from "semantic-ui-react";
+import React, { useEffect, useState } from "react"
+import PokemonCollection from "./PokemonCollection"
+import PokemonForm from "./PokemonForm"
+import Search from "./Search"
+import { Container } from "semantic-ui-react"
 
 function PokemonPage() {
+  const [triggerFetch, setTriggerFetch] = useState(false)
+  const [pokemonList, setPokemonList] = useState(null)
+  const [searchedPokemon, setSearchedPokemon] = useState("")
```

Comment:
Going back to this search idea, instead of storing a separate copy of searched Pokemon, you could store the `serach` state here and just handle the filtering in this component.